Quests/Bonus Quests/quest_00150000.csv

```diff
@@ -0,0 +1,3 @@
+Title#0,"""Mission Speciale: Tokyo [Gold]"""
+Explanation#0,"""Sur la planete Terre, nous sommes parvenus<br>a contenir certains ennemis<br>dans une section de Tokyo.<br>De plus, les forces ennemies<br>se sont beaucoup affaiblies.<br>Ne ratez pas cette chance. Rendez-vous vite dans<br>cette zone, et battez les ennemis!<br>―――――――――――――――――――<br>Start: Utilisez Clé Bonus Tokyo [Gold]<br>Objectif: Battez tous les ennemis avant la fin du temps."""
+SimpleExplanation#0,"""Battez tous les ennemis !"""
```

Comment:
Should there be a space between `ennemis` and `!`?

Reply:
corrected

Reply:
Not necessary, I change that